### ISSUE TITLE
Attaching data

### ISSUE THREAD

**Issue by dereklieu** [developmentseed/nethope-sitrep-api#3]
For data pulled from external API's like HDX, and for sources we host ourselves, how should these be attached to reports?

@drewbo interested in hearing your thoughts here. Sean Gorman at DG also has some experience with this, and they use notebooks internally and have thought about this. I was thinking to set up a call with them next week to hear more about their approach.

**Comment by drewbo** [comment]
@dereklieu yeah let's set up a call. If it's something that we can handle easily with an external `requests` call, that's probably the best way. If it's larger, then it gets tricky
